### DISCUSSION TITLE
How to disable comments on a particular site?

### DISCUSSION THREAD

**Discussion by MicPec** [dillonzq/LoveIt#810] [Q&A]
Hi, I'm using [giscus](https://giscus.app/) to provide comments on my blog, but I need to disable it on particular sites eg. About

BTW. Many thanks for the awesome theme. You are great!

**Comment by owl4ce** [comment]
> Hi, I'm using [giscus](https://giscus.app/) to provide comments on my blog, but I need to disable it on particular sites eg. About
> 
> BTW. Many thanks for the awesome theme. You are great!

https://hugoloveit.com/theme-documentation-content/#front-matter

**Reply by owl4ce** [reply]
```cfg
43 comment:
44   enable: false
45   # ...
```

**Comment by MicPec** [comment]
Thanks, I looked at it but didn't see it.
